tests/system/robot/chromeTests.py

```diff
@@ -42,38 +43,71 @@ def checkbox_labelled_by_inner_element():
 
 
 def announce_list_item_when_moving_by_word_or_character():
+	spy = _nvdaLib.getSpyLib()
 	_chrome.prepareChrome(
 		r"""
-			<ul style="list-style-type:none">
-				<li>xy</li>
-				<li>z</li>
-			</ul>
+			<div contenteditable="true">
+				<p>Before list</p>
+				<ul style="list-style-type:none">
+					<li>small cat</li>
+					<li>big dog</li>
+				</ul>
+			</div>
 		"""
 	)
+	# Move focus into the content editable.
+	# This will turn on focus mode.
+	# And the caret will be on the paragraph before the list.
+	spy.emulateKeyPress("tab")

```

Comment:
I wonder if we should use the option to speak focus/browse mode and check for it here?